galite-core/src/main/kotlin/org/kopi/galite/visual/ui/vaadin/field/TextFieldNavigationHandler.kt

```diff
@@ -105,18 +105,6 @@ open class TextFieldNavigationHandler protected constructor(private val isMulti:
     addKeyNavigator(field, Key.ARROW_UP, KeyModifier.of("Control")) {
       field.connector.firePreviousEntry()
     }
-    if (!isMulti) {
-      // In multiline fields these keys are used for other stuff
-      addKeyNavigator(field, Key.ARROW_UP) {
-        field.fieldConnector.gotoPrevField()
-      }
-      addKeyNavigator(field, Key.ARROW_DOWN) {
-        field.fieldConnector.gotoNextField()
-      }
-      addKeyNavigator(field, Key.ENTER) {
-        field.fieldConnector.gotoNextField()
-      }
-    }
```

Comment:
It is not prefered to desactivate the arrow up / arrow down and enter key actions for the single block. Instead of deleting these lines, change the if condition to (!isMulti && field !is VCodeField)